### PR TITLE
New sandbox:ips call

### PR DESCRIPTION
**Pull request by EvilJordan**
Added standalone `sandbox:ips` CLI call to list both inbound **AND** outbound IPs with optional JSON output.

When requesting JSON output, resultant output wraps in key for easy parsing:
```
{"inbound":["3.210.241.243","3.212.83.232","18.206.49.158","34.203.78.166","3.208.56.208","54.147.234.135"]}
{"outbound":["3.220.190.116","34.195.33.38","3.218.239.11"]}
```

### CODE REVIEW

**Comment by Gektorian** [comment]
Hi @EvilJordan, do you think we could also update Readme.MD with a new endpoint and example of usage for how it is for other endpoints?

File: README.md [hunk 1]
```diff
@@ -285,6 +285,7 @@ Use `sfcc-ci --help` or just `sfcc-ci` to get started and see the full list of c
     sandbox:realm:list [options]                                    List realms eligible to manage sandboxes for
     sandbox:realm:update [options]                                  Update realm settings
     sandbox:list [options]                                          List all available sandboxes
+    sandbox:ips [options]                                           List inbound and outbound ips for sandboxes
```
Comment (sholsinger) [inline README.md:288]:
```suggestion
    sandbox:ips [options]                                           List inbound and outbound IP addresses for sandboxes
```

File: cli.js [hunk 2]
```diff
@@ -264,6 +264,22 @@ program
         console.log();
     });
 
+program
+    .command('sandbox:ips')
+    .description('List inbound and outbound ips for sandboxes')
```
Comment (sholsinger) [inline cli.js:269]:
```suggestion
    .description('List inbound and outbound IP addresses for sandboxes')
```

**Review by sholsinger — CHANGES_REQUESTED** [review]
Thanks for your awesome contribution Jordan. I'd not assume that 'ips' in that context would be 100% clear to all users. :)

**Review by sholsinger — APPROVED** [review]

**Comment by hnestmann** [comment]
Thanks for your contribution @EvilJordan I really love that you guys participate. Do I see this right that  this filters information from the sandbox:get call? If that is the case, would you think that sandbox:get:ips  would be an idea too? We could have other useful lookups such  as sandbox:get:hostname

**Comment by EvilJordan** [comment]
@hnestmann This is a standalone call that draws from (and slightly extends) the existent `printInboundIPs()` method that seems to only be called during the `sandbox:alias:list` and `sandbox:alias:add` methods...

I'm all for having parity of `sfcc-ci` with the available API methods as listed in Swagger! The more, the better!

**Comment by hnestmann** [comment]
@EvilJordan Cool, I see. Let's wait @tobiaslohr s thoughts :)

**Comment by EvilJordan** [comment]
Bumping this... just had a painful client call trying to walk them through gaining access to the Swagger UI, when all they had used before was sfcc-ci, in order to retrieve IPs.

I think this would be a very useful addition to the toolkit!

**Review by Gektorian — APPROVED** [review]

**Review by tobiaslohr — APPROVED** [review]

**Comment by tobiaslohr** [comment]
Thanks a lot @EvilJordan for contributing! Sorry for the delay, we had to fix a CircleCI related build issue first.

**Pull request merged**